tests/isolation-controls-binding.test.js

```
@@ -0,0 +1,161 @@
+
+function loadOrchestrator() {
+  // Reset DOM and globals for each test
+  document.body.innerHTML = ISOLATION_HTML;
+  window.AudioContext       = MockAudioContext;
+  window.webkitAudioContext = MockAudioContext;
+  window.AudioWorkletNode   = class { constructor() { this.port = { postMessage: () => {} }; } connect(){} disconnect(){} };
+  window.Worker             = MockWorker;
+  window.SharedArrayBuffer  = SharedArrayBuffer;
+
+  // Suppress the bootstrap polling timer
+  const realSetInterval = window.setInterval;
```

ESLint

[error] 'window' is not defined.

(no-undef)